decidim-assemblies/lib/decidim/assemblies/content_blocks/registry_manager.rb

```diff
@@ -82,7 +82,7 @@ def self.register!
             content_block.public_name_key = "decidim.content_blocks.last_activity.name"
             content_block.settings_form_cell = "decidim/content_blocks/last_activity_settings_form"
             content_block.settings do |settings|
-              settings.attribute :max_last_activity_users, type: :integer, default: Decidim::ContentBlocks::ParticipatorySpaceLastActivityCell::DEFAULT_MAX_LAST_ACTIVITY_USERS
+              settings.attribute :max_last_activity_users, type: :integer, default: Decidim.default_max_last_activity_users
```

Comment:
I had a problem with requiring/using classes defined in `app/` from `lib/`. Moving this constant to a `config_accessor` was a quick way of solving this problem.